### DISCUSSION TITLE
Ubuntu 22.04 or 24.04 spins

### DISCUSSION THREAD

**Discussion by farhatizakaria** [regolith-linux/regolith-desktop#1093] [General]
I found out Regolith 1.6.0 hirsute, it's good distro, but it's outdated. Why there's no Ubuntu 22.04 or 24.04 spin available ? is the project meant only to ship a desktop only ? just curious because the experience of this desktop installed with fresh ISO (installation) is much better than installed alongside another desktop. 😅

**Comment by farhatizakaria** [comment]
I've read https://regolith-desktop.com/posts/status_update_08_24/ for knowing the status of 24.04 iso

**Reply by kgilmer** [reply]
Hi @farhatizakaria , yeah it would be great if we could continue producing the ISO.  With limited time, we can only work on a few things on an ongoing basis, and the ISO has been very time intensive for a few reasons.  Our current plan is to publish Regolith packages upstream into Debian and Ubuntu and then produce an Ubuntu variant that is more officially supported.  This is a larger term project though.  @SoumyaRanjanPatnaik has been working on this recently.
